### PR TITLE
Adjust README

### PR DESCRIPTION
**Pull request by pwalczysko**
Tested the README workflow

Adjusted:

 - add install instructions link to napari
 - remark on the envs
 - fix a bug of the python workflow
 - remove the ``0`` subfolder opening routine, as it crashes atm

cc @jburel @will-moore

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,35 +24,36 @@ https://napari.org/docs/plugins/index.html
...
-Napari will use `ome-zarr` to open images that the plugin recognises as ome-zarr.
+Napari will use `napari-ome-zarr` plugin to open images that the plugin recognises as ome-zarr.
 The image metadata from OMERO will be used to set channel names and rendering settings
 in napari::
 
-    $ napari "https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.3/9836842.zarr/"
+    napari "https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.3/9836842.zarr/"
+
 
+If a dialog in napari pops up, encouraging you to choose a reader, choose ``napari-ome-zarr`` and click OK.
```
Comment (will-moore) [inline README.md:42]: Could mention that you can specify the plugin on cli with `napari --plugin napari-ome-zarr "https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.3/9836842.zarr/"`

File: README.md [hunk 2]
```diff
@@ -24,35 +24,36 @@ https://napari.org/docs/plugins/index.html
...
 
     import napari
 
     viewer = napari.Viewer()
-    viewer.open('https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.3/9836842.zarr/')
+    viewer.open("https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.4/idr0101A/13457537.zarr", plugin="napari-ome-zarr")
 
     napari.run()
 
-If single zarray is passed to the plugin, it will be opened without the use of
```
Comment (will-moore) [inline README.md:52]: When I tried this, I didn't need the `napari.run()` at least in a python terminal

File: README.md (hunk 1 above)
Comment (pwalczysko) [reply README.md:42]: But, confusingly, when opening from S3, napari is not confused about which plugin to use.
Whereas locally, it is different, such as

```
napari --plugin napari-ome-zarr '6001240.zarr'
``` 

Does not produce a dialog, whereas

```
napari  '6001240.zarr'
```

Produces 

Screenshot 2022-11-03 at 11 09 57

Comment (pwalczysko) [reply README.md:42]: fixed in https://github.com/ome/napari-ome-zarr/pull/72/commits/509461c0e944f6f12d62296e66a3977785bdc254

Comment (will-moore) [reply README.md:42]: I don't know why the difference when the file is local. But you say "If a dialog in napari pops up, encouraging you to choose a reader, choose ``napari-ome-zarr`` and click OK."
If that happens, you can stop it happening with `--plugin napari-ome-zarr`, regardless of whether the file is local or not. So I would mention `--plugin napari-ome-zarr` option before mentioning local usage

File: README.md (hunk 2 above)
Comment (pwalczysko) [reply README.md:52]: Yes, can confirm this in python terminal. But if you leave it out in the script, then napari opens and immediately closes. With the ``napari.run()`` in the script, napari persists and is usable.

File: README.md (hunk 1 above)
Comment (pwalczysko) [reply README.md]: fixed in https://github.com/ome/napari-ome-zarr/pull/72/commits/2680af2f0923d5b861db0f6a38c4a866657d0f52

**Review by will-moore — APPROVED** [review]

**Pull request merged**